### PR TITLE
Modernize travis-ci config

### PR DESCRIPTION
**Pull request by dfreedm**
- polymer-cli
- docker instances for faster booting
- chrome addon instead of apt

### CODE REVIEW

File: cleanup-passes/travis.ts [hunk 1]
```diff
@@ -18,25 +18,24 @@ import * as fs from 'fs';
...
-type TravisEnv = {global?: string[], matrix?: string[]};
+type TravisEnv = {
+  global?: string[],
+  matrix?: string[]
+};
 
 interface TravisConfig {
   before_script?: string[];
   addons?: {
     firefox?: string | number,
```
Comment (rictic) [inline cleanup-passes/travis.ts]: Prefer semicolons in interfaces

File: cleanup-passes/travis.ts [hunk 2]
```diff
@@ -75,7 +64,8 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
 
   // use ubuntu trusty
   travis.dist = 'trusty';
-  travis.sudo = 'required';
+  // use docker
+  travis.sudo = 'false';
```
Comment (rictic) [inline cleanup-passes/travis.ts]: This can just be left off then, yeah?

File: cleanup-passes/travis.ts [hunk 3]
```diff
@@ -18,25 +18,24 @@ import * as fs from 'fs';
 import * as yaml from 'js-yaml';
 import * as path from 'path';
 
-import {register} from '../cleanup-pass';
-import {ElementRepo} from '../element-repo';
-import {existsSync, makeCommit} from './util';
+import { register } from '../cleanup-pass';
+import { ElementRepo } from '../element-repo';
+import { existsSync, makeCommit } from './util';
```
Comment (rictic) [inline cleanup-passes/travis.ts]: `npm run format` if that command exists plz

otherwise don't worry about it

**Review by rictic — APPROVED** [review]

File: cleanup-passes/travis.ts (hunk 2 above)
Comment (dfreedm) [reply cleanup-passes/travis.ts]: No it has to be explicitly set to `'false'` to use docker instances.

File: cleanup-passes/travis.ts (hunk 3 above)
Comment (dfreedm) [reply cleanup-passes/travis.ts]: Ah this was VS Code's doing.

Comment (dfreedm) [reply cleanup-passes/travis.ts]: No `format` script. Should I port over the `clang-format` setup from `polymer-cli`?

**Pull request merged**